src/Drupal/DrupalAutoloader.php

```diff
@@ -18,70 +17,7 @@
  *
  * Code re-used from https://github.com/mglaman/phpstan-drupal/blob/main/src/Drupal/DrupalAutoloader.php.
  */
-class DrupalAutoloader {
```

Comment:
As Code Quality tools runs the same rules on itself and we haven't had complexity check like a year, we had this class that was too complex :slightly_smiling_face: I've now split it up into 2 classes, moved most of the method to new DrupalAutoloaderBase.php class.

And if you ask why we have this class in the first place, it's because to mimic how Drupal loads it's codebase for Psalm scanner. Drupal does this during runtime. This class basically just includes lot of files..